src/trumbowyg.js

```diff
@@ -1668,11 +1668,30 @@ Object.defineProperty(jQuery.trumbowyg, 'defaultOptions', {
                     return prop + '="' + a[prop] + '"';
                 }).join(' ');
 
-                html += '<label><input type="' + (field.type || 'text') + '" name="' + n + '"' +
-                    (field.type === 'checkbox' && field.value ? ' checked="checked"' : ' value="' + (field.value || '').replace(/"/g, '&quot;')) +
-                    '"' + attr + '><span class="' + prefix + 'input-infos"><span>' +
-                    (lg[l] ? lg[l] : l) +
-                    '</span></span></label>';
+                if (typeof field.type === 'function') {
+                  html += field.type(field, prefix, lg);
+                } else {
+                  html += `
+                    <div class="${prefix}input-row">
+                      <div class="${prefix}input-infos">
+                        <label for="form-${fieldName}">
+                          <span>${lg[l] ? lg[l] : l}</span>
+                        </label>
+                      </div>
+                      <div class="${prefix}input-html">
+                        <input
+                          type="${field.type || 'text'}"
+                          name="${n}"
+                          id="form-${fieldName}"
+                          ${attr ? ' '+attr : ''}
+                          ${field.type === 'checkbox' && field.value ? ' checked="checked"' : ''}
+                          value="${field.value || ''}"
+                        />
+                      </div>
+                    </div>
+                  `;
```

Comment:
Not supported in older browsers :/
Also, add some bytes (I did not check how many)

Reply:
No problems, will update.
I have found this a lot cleaner to read than the old way, but if you still support it, easy as